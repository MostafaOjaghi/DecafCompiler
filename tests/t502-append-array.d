
void main() {
    int[] a;
    int[] b;
    int[] c;
    int i;

    a = NewArray(5, int);
    for (i = 0; i < 5; i = i + 1) {
        a[i] = i + 50;
    }
    b = NewArray(6, int);
    for (i = 0; i < 6; i = i + 1) {
        b[i] = i + 40;
    }
    for (i = 0; i < 5; i = i + 1) {
        Print(a[i]);
    }
    for (i = 0; i < 6; i = i + 1) {
        Print(b[i]);
    }
    c = a + b;
    for (i = 0; i < 11; i = i + 1) {
        Print(c[i]);
    }
}