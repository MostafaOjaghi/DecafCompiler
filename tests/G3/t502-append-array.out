50
51
52
53
54
40
41
42
43
44
45
50
51
52
53
54
40
41
42
43
44
45